application/common/config/main.php

```diff
@@ -219,5 +219,6 @@
             Env::getArrayFromPrefix('METHOD_')
         ),
         'mfaLifetime' => Env::get('MFA_LIFETIME', '+2 hours'),
+        'contingentUserDuration' => Env::get('CONTINGENT_USER_DURATION', '+4 weeks'),
```

Comment:
thank you for giving a default!  Do you mention this new env var in the docs or README?

Reply:
If `local.env.dist` counts as a doc, yes. I just checked and `README.md` doesn't mention env vars.